qsr_lib/cfg/defaults.ini

```diff
@@ -1,2 +1,3 @@
 [arg_relations_distance]
-relations_and_values = {"0": 5., "1": 15., "2": 120.}
+;relations_and_values = {"0": 5., "1": 15., "2": 120.}
+relations_and_values = {}
```

Comment:
Yes I know.... But, I would leave it so that to remind me and a potential user that this should be a dictionary in the case of a custom ini file... Will probably move it in the documentation but I would need to change something in the code also.